tests/_support/AcceptanceTester.php

```diff
@@ -12,15 +12,423 @@
  * @method void am($role)
  * @method void lookForwardTo($achieveValue)
  * @method void comment($description)
- * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = NULL)
+ * @method \Codeception\Lib\Friend haveFriend($name, $actorClass = null)
  *
  * @SuppressWarnings(PHPMD)
-*/
+ */
 class AcceptanceTester extends \Codeception\Actor
 {
-    use _generated\AcceptanceTesterActions;
+	use _generated\AcceptanceTesterActions;
 
-   /**
-    * Define custom actions here
-    */
+	/**
+	 * Install joomla CMS
+	 *
+	 * @Given Joomla CMS is installed
+	 */
+	public function joomlaCMSIsInstalled()
+	{
+		// throw new \Codeception\Exception\Incomplete("Step `Joomla CMS is installed` is not defined");
+	}
+
+	/**
+	 * @When Login into Joomla administrator with username :arg1 and password :arg1
+	 */
+	public function loginIntoJoomlaAdministrator($username, $password)
+	{
+		$I = $this;
+		$I->doAdministratorLogin($username, $password);
```

Comment:
Instead of using JoomlaBrowser method, implement the method yourself, something:

```php
 $I->amOnPage('/administrator/index.php');
$I->waitForElement(['id' => 'mod-login-username'], 60);
...
```

on the long run we want to not have dependencies with JoomlaBrowser.